modules/client-configuration/pages/vhm-aws.adoc

```diff
@@ -0,0 +1,43 @@
+[[vhm-aws]]
+= VHM and Amazon Web Services
+
+You can use a {productname} VHM to manage Amazon Web Services (AWS) clusters.
+
+The VHM allows {productname} to obtain and report information about your clusters, including the ability to do CVE audits, and report if containers are outdated.
```

Comment:
Why are we mentioning containers here? In any case, using a VHM does not enable you to do CVE audits or getting info about patches. You must have those instances also registered as systems into SUSE Manager in order to manage them.